aws/resource_aws_ses_configuration_set_test.go

```diff
@@ -77,16 +51,18 @@ func testAccCheckAwsSESConfigurationSetExists(n string) resource.TestCheckFunc {
 
 		conn := testAccProvider.Meta().(*AWSClient).sesConn
 
-		response, err := conn.ListConfigurationSets(&ses.ListConfigurationSetsInput{})
+		response, err := conn.DescribeConfigurationSet(&ses.DescribeConfigurationSetInput{
+			ConfigurationSetName: aws.String(rs.Primary.ID),
+		})
+
 		if err != nil {
 			return err
 		}
 
 		found := false
-		for _, element := range response.ConfigurationSets {
-			if *element.Name == fmt.Sprintf("some-configuration-set-%d", escRandomInteger) {
-				found = true
-			}
+
+		if *response.ConfigurationSet.Name == rs.Primary.ID {
```

Comment:
we can safely extract this `Name` variable with the AWS SDK's conversion: `aws.StringValue()`

Reply:
in addition, the equality logic could be updated to return immediately if the condition is satisfied e.g.
```
if aws.StringValue(response.ConfigurationSet.Name) != rs.Primary.ID {
    return fmt.Errorf("The configuration set was not created")
}
return nil
````